frontend/src/components/pages/FridgeManagement/FridgeCheckIns/DeleteCheckIns.tsx

```diff
@@ -0,0 +1,145 @@
+    }
+    setDateRangeError(newError);
+    return valid;
+  };
+
+  const openModal = () => {
+    if (!validateForm()) {
+      return;
+    }
+
+    onOpen();
+  };
+  const onDeleteClick = async () => {
+    if (!validateForm()) {
+      return;
+    }
+    const res = await CheckInAPIClient.deleteCheckInsByDateRange(
+      dateRange[0].toString(),
+      endOfDay(dateRange[1].toDate()).toString(),
+    );
+    if (!res) {
+      toast({
+        title: "There are no check-ins in this date range.",
+        status: "error",
+        duration: 7000,
+        isClosable: true,
+      });
+      return;
+    }
+    toast({
+      title: "Check-ins have been successfully deleted",
+      status: "success",
+      duration: 7000,
+      isClosable: true,
+    });
+    history.push(Routes.ADMIN_CHECK_INS);
+  };
+
+  return (
+    <Container variant="responsiveContainer">
+      <Text textStyle="mobileHeader2" mt="2em">
+        Delete shifts
+      </Text>
+      <FormControl isRequired isInvalid={!!dateRangeError} m="3em 0">
+        <FormLabel fontWeight="600">Select date range</FormLabel>
+        <FormHelperText mb="1em">Delete all shifts between:</FormHelperText>
+        <DatePicker
+          range
+          minDate={new Date()}
+          value={null}
+          onChange={(e: DateObject[]) => {
+            setDateRange(e);
+            setDateRangeError("");
+          }}
+          render={(
+            value: string,
+            openCalendar: React.MouseEventHandler<HTMLInputElement>,
+          ) => {
+            return (
+              <HStack maxW="740px" spacing="1rem">
+                <Input
+                  onClick={openCalendar}
+                  value={new DateObject(value[0]).format("MMM DD, YYYY")}
+                />
+                <Text>to</Text>
+                <Input
+                  value={
+                    dateRange[1]
+                      ? new DateObject(dateRange[1]).format("MMM DD, YYYY")
+                      : "MM/DD/YYYY"
+                  }
+                  disabled
+                />
+              </HStack>
+            );
+          }}
+        />
+        <FormErrorMessage>{dateRangeError}</FormErrorMessage>
+      </FormControl>
+      <Button onClick={openModal} variant="navigation">
+        Delete shifts
+      </Button>
+      <GeneralDeleteShiftModal
+        title="Cancel shifts in range"
+        bodyText={
+          dateRange[1] &&
+          `Are you sure you want to cancel these fridge check-in shifts? This will remove all shifts ${
+            dateRange[0].toString() === dateRange[1].toString()
+              ? ` on ${dateRange[0].format("MMMM D").toString()} `
+              : ` between ${dateRange[0]
+                  .format("MMMM D")
+                  .toString()} and ${dateRange[1].format("MMMM D").toString()} `
+          }
+        and notify any assigned volunteers.`
+        }
+        buttonLabel="Cancel shifts"
```

Comment:
Lets change the wording to "Delete shifts". I know the designs say "Cancel", but i talked with Amy and it should be "Delete shifts" cuz the screen before had "Delete shifts" wording, so just to be consistent.